test/widgets/message_list_test.dart

```diff
@@ -484,6 +487,22 @@ void main() {
 
       debugNetworkImageHttpClientProvider = null;
     });
+
+    testWidgets('Bot user is distinguished by showing an icon', (tester) async {
+      prepareBoringImageHttpClient();
```

Comment:
In the test before this test, I noticed that there was a fake `HttpClient` set, that could be replaced by calling this method that was factored out in [#590](https://github.com/zulip/zulip-flutter/pull/590/commits/4ab1a82ebf5f71ba27413fded606af075c871b50).

```diff
testWidgets('Updates avatar on RealmUserUpdateEvent', (tester) async {

-  final httpClient = FakeImageHttpClient();
-    debugNetworkImageHttpClientProvider = () => httpClient;
-    httpClient.request.response
-      ..statusCode = HttpStatus.ok
-      ..content = kSolidBlueAvatar;

+  prepareBoringImageHttpClient();
  
}
```

Is it ok if I add an additional [NFC] commit to this PR with the requested change, or should we just ignore it for now?

Reply:
Seems reasonable to me! 🙂